www/src/html.js

```diff
@@ -64,6 +64,10 @@ export default class HTML extends React.Component {
           {this.props.postBodyComponents}
           <link
             rel="preload"
+            href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css"
+            as="style"
+          />
```

Comment:
Almost there! This `preload` link (and only this one) should go right after the opening `<head>` tag.